cherry_picker/cherry_picker.py

```diff
@@ -245,6 +245,9 @@ def continue_cherry_pick(self):
 
             self.cleanup_branch(cherry_pick_branch)
 
+            click.echo("\nBackport PR:\n")
+            click.echo(updated_commit_message)
```

Comment:
So is this printing out the title, message body, or both? The previous line is ambiguous without context so I don't know what to expect to see in the eventual output.